lib/rules/logical-assignment-operators.js

```diff
@@ -112,7 +112,7 @@ function isBooleanCast(expression, scope) {
 /**
  * Returns true for:
  * truthiness checks:  value, Boolean(value), !!value
- * falsyness checks:   !value, !Boolean(value)
+ * falsiness checks:   !value, !Boolean(value)
```

Comment:
```suggestion
 * falseness checks:   !value, !Boolean(value)
```

Reply:
That seems a bit misleading- the point of the name falsy is that it's not exactly false. Also, if we go with "falseness", we would have to change "truthiness" to "truthness", right?

Reply:
I haven't seen `falsiness` word being used so I suggested `falseness` if it is used then I am ok with the change. Can you pls check and confirm this?

Reply:
https://en.m.wiktionary.org/wiki/falsiness